core/modules/widgets/navigator.js

```diff
@@ -557,10 +557,12 @@ NavigatorWidget.prototype.handleImportTiddlersEvent = function(event) {
 NavigatorWidget.prototype.handlePerformImportEvent = function(event) {
 	var self = this,
 		importTiddler = this.wiki.getTiddler(event.param),
-		importData = this.wiki.getTiddlerDataCached(event.param,{tiddlers: {}}),
+		importData,
 		importReport = [];
-	// Add the tiddlers to the store
 	importReport.push($tw.language.getString("Import/Imported/Hint") + "\n");
+	// If you need to modify the import tiddler payload, consider th-importing-tiddler instead!
```

Comment:
I'm sorry to keep raising the same issue, but the exclamation mark is not needed here. In English it would also be clearer to replace the comma with the word "then"

Reply:
Will be updated